lib/default-config.js

```diff
@@ -7,9 +7,9 @@ const DEFAULT_CONFIG = Object.freeze({
   'no-changes-template': `* No changes`,
   'version-template': `$MAJOR.$MINOR.$PATCH`,
   'version-resolver': {
-    major: { labels: ['major'] },
-    minor: { labels: ['minor'] },
-    patch: { labels: ['patch'] },
+    major: { labels: [] },
+    minor: { labels: [] },
+    patch: { labels: [] },
```

Comment:
Hmm already optional? Seems the latest commit has now made it optional, unless I am mistaken @darkdreamingdan

Reply:
The RESOLVED_VERSION variable means it was always optional to use the version-resolver, because unless you use the RESOLVED_VERSION template variable in your config.yml then the version won't increment.  

The above change is not related to making version resolver optional.  Because the default config does a deep merge, i had to remove the default labels of 'major', 'minor', 'patch', because it is impossible to remove them if they are in the default config.  I noticed this within unit tests after i updated them.  So by default, $RESOLVED_VERSION will just increment the patch version unless you provide label config.  This is now fixed.

Reply:
Good thing we have unit test 🗡️